### PR TITLE
biological_replicates and technical_replicates

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
**What problem is the PR solving / What's new?**
Since I was the one originally opposing the long names, it only seems fair I do this one :yum: 

**Checklist**
- [x] I made a PR to develop (not master)
- [x] If applicable: I updated the docs
- [x] I updated the CHANGELOG
- [x] These changes are covered by the tests

### CODE REVIEW

**Comment by Maarten-vd-Sande** [comment]
@siebrenf this is ready. Maybe we should keep this here a bit longer? We can release it maybe together with rerunning and scRNA? Or too much big changes at the same time, so if bugs appear we won't know what is causing it?

File: seq2science/rules/peak_counts.smk [hunk 1]
```diff
@@ -140,7 +140,7 @@ def get_coverage_table_replicates(file_ext):
 
 rule coverage_table:
     """
-    Use gimmemotif's coverage_table to generate a cpunt table with the nr of reads
+    Use gimmemotif's coverage_table to generate a count table with the nr of reads
```
Comment (siebrenf) [inline seq2science/rules/peak_counts.smk:143]: 🎉

**Review by siebrenf — CHANGES_REQUESTED** [review]
scripts still need some attention. sec.

**Review by siebrenf — APPROVED** [review]
I think that's it.

Yea, we should run a few more test runs with this. I'll rerun Angela's (RNA-seq & ATAC-seq) data to compare it! (she should have it ~ next week)

**Comment by Maarten-vd-Sande** [comment]
thanks for the catches, I really though I had them all (those names are everywhere :yum: )

**Pull request merged**